### ISSUE TITLE
ES6 module?

### ISSUE THREAD

**Issue by derek-zhou** [jitbit/HtmlSanitizer#12]
It will be great if this is published as a ES6 module, with proper exports, so I can just do:
``` javascript
import HtmlSanitizer from '@jitbit/htmlsanitizer'
```

**Comment by doolijb** [comment]
Agreed. I actually got started on this, but when I went to convert the tests to QUnit, I realized that this entire library basically has to be rewritten.

**Comment by alex-jitbit** [comment]
I'm trying to figure out is there's a cross-browser compatible way to export a module

**Comment by simialbi** [comment]
@derek-zhou https://github.com/simialbi/html-sanitizer

**Comment by thdoan** [comment]
An ES6 module that works with TypeScript would be nice.

**Comment by juliovedovatto** [comment]
Thanks for the recent update, @alex-jitbit 🚀

I’m currently preparing for the Vue.js Senior certification exam, and one of the exercises requires client-side input sanitization and recommended to use your package. I just installed your package directly from this repo, and it worked perfectly. Appreciate the solid work!
